responsibleai/tests/model_analysis/test_model_analysis_validations.py

```diff
@@ -30,7 +30,8 @@ def test_validate_unsupported_task_type(self):
         X_test['target'] = y_test
 
         message = ("Unsupported task type 'regre'. "
```

Comment:
I don't think we should be seeing this error - this is for tabular data only - this validation should not be run for vision?

Reply:
Update: tweaked the test case to reflect the original prod user error message